cmds/exp/cpu/cpu.go

```diff
@@ -18,19 +18,22 @@ import (
 	"path/filepath"
 	"strings"
 	"syscall"
+	"time"
 	"unsafe"
 
 	// We use this ssh because it implements port redirection.
 	// It can not, however, unpack password-protected keys yet.
 	"github.com/gliderlabs/ssh"
 	"github.com/kr/pty" // TODO: get rid of krpty
-	"github.com/u-root/u-root/pkg/mount"
 	"github.com/u-root/u-root/pkg/termios"
 	// We use this ssh because it can unpack password-protected private keys.
 	ossh "golang.org/x/crypto/ssh"
 	"golang.org/x/sys/unix"
 )
 
+// a nonce is a [32]byte containing only printable characters, suitable for use as a string
```

Comment:
Based on the `getNonce` function, this should always be hex characters [0-9a-f]. Perhaps it should be mentioned here?

Reply:
I don't want to assume that but thanks.